dev-libs/jansson/jansson-2.10.ebuild

```diff
@@ -37,3 +37,9 @@ multilib_src_compile() {
 		HTML_DOCS=( "${BUILD_DIR}"/doc/_build/html/. )
 	fi
 }
+
+multilib_src_install() {
```

Comment:
Please don't do such potentially-breaking changes in stable ebuilds. Instead, let's limit it ot 2.11.